addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -84,6 +84,7 @@ function factory(dependencies) {
         unfold() {
             if (this.thread) {
                 this.thread.update({ pendingFoldState: 'open' });
+                this.threadViewer.addComponentHint('chat-window-unfolded');
```

Comment:
I don't like the `unfolded` name, maybe we could start to use `open` instead, similar to python value?

Reply:
opened* with proper conjugaison

Reply:
python uses 3 states: `open`/`folded`/`closed`.
This actually mixes fold state and minimize state of some sort.
So I prefer `(un)fold(ed)` than `open(ed)/closed`.

Reply:
Having both `is_minimized` and `fold_state` variables are mistakes from the python that I'd like to get rid of eventually. They are not even sync by a compute method, they are currently written and saved in parallel with all the inconsistencies that can happen if one is written without the other...

I think there are indeed 3 "chat window"/thread states for a given user, for which the ideal names are: opened, closed, minimized (= folded).

Reply:
Having just 3 chat window/thread states at the end is great, but I wouldn't do make partial changes with this idea right now. We will have super annoying inconsistencies everywhere, because all these new words clash with old ones:

- (new) "opened" = (old) is_minimized: true & fold_state: "open"
- (new) "closed" = (old) is_minimized: false (mostly fold_state: "closed", but that's not even true all the time...)
- (new) "minimized" = (old) is_minimized: true & fold_state: "folded"

This is very unsafe to make partial changes, so let's not do that right now and instead use wording that are not confusing at all in the meantime. Here "unfolded" is a better fit.